# Bepaling van warmtecapaciteit van een onbekend materiaal


# Introductie

Onbekende materialen kunnen geïdentificeerd worden door hun eigenschappen te meten. Een van deze eigenschappen is de warmtecapaciteit. In dit practicum gaan we de warmtecapaciteit van een onbekend materiaal bepalen door middel van een calorimeter experiment. Daarbij wordt een bepaalde massa van het materiaal naar een bekende temperatuur gebracht waarna het in een bekende hoeveelheid water met bekende temperatuur wordt geplaatst. Door de temperatuur van het water te meten na het mengen kan de warmtecapaciteit van het onbekende materiaal worden berekend.

# Theorie

De soortelijke warmte $c$ van een materiaal is gedefinieerd als de hoeveelheid warmte $Q$ die nodig is om de temperatuur $T$ van een kilogram van het materiaal met één graad Celsius (of één Kelvin) te verhogen:

$$
    c = \frac{Q}{m \Delta T}
$$ (eq:heat_capacity)

Waarbij $Q$ de hoeveelheid warmte in Joules is, $m$ de massa in kilogram is en $\Delta T$ de verandering in temperatuur is. Gegeven de wet van Black, die stelt dat de totale hoeveelheid warmte in een geïsoleerd systeem constant blijft, kunnen we de warmte die het onbekende materiaal verliest gelijkstellen aan de warmte die het water opneemt:

$$
    Q_{materiaal} = -Q_{water} 
$$ (eq:black)

wanneer we de massa's en de begintemperaturen van beide systemen kennen, maar slechts een van de twee soortelijke warmtes, kunnen we de onbekende soortelijke warmte berekenen. We combineren vergelijkingen [](#eq:heat_capacity) en [](#eq:black) om de volgende vergelijking te krijgen:

$$
    T_e = \frac{c_w m_w T_{w,b}+c_m m_m T_{m,b}}{c_w m_w + c_m m_m}
$$ (eq:combined)

Waarbij de subscripts $b$ en $e$ respectievelijk staan voor begintoestand en eindtoestand, $w$ voor water en $m$ voor het onbekende materiaal.

Bij metingen aan verschillende massa's van het onbekende materiaal en vervolgens een least square fit aan bovenstaande vergelijking kunnen we een precieze waarde voor de soortelijke warmte van het onbekende materiaal bepalen. Dat is, wanneer de warmtecapaciteit van bijvoorbeeld de beker te verwaarlozen is.


# Methode en materialen

## Ontwerp
De bovenstaande theorie wordt gebruikt om de soortelijke warmte van een onbekend materiaal te bepalen. Het experiment bestaat uit het verwarmen van verschillende massa's van het onbekende materiaal tot een bekende temperatuur, waarna het in een bekende hoeveelheid water met bekende temperatuur wordt geplaats. Door de temperatuur van het water te meten na het mengen kan de warmtecapaciteit van het onbekende materiaal worden berekend. Om de tijd voor het meten van meerdere materialen te reduceren, worden de data van de verschillende groepen in het lokaal samengevoegd. Van tevoren is afgesproken welke massa's door welke groep worden gemeten, en hoeveel water er gebruikt wordt.

## Materialen
Hieronder staat de lijst van benodigde materialen bij deze proef:
- Calorimeter
- Thermometer of temperatuursensor
- Verwarmingsbron 
- Diverse massablokjes van onbekend materiaal
- Weegschaal
- Water
- Maatcilinder of maatbeker


```{figure} figures/c_onbekend_setup.jpg
:width: 70%
:label: fig_c_onbekend_setup

Een schematische weergave van de opstelling
```


## Procedure
Bespreek wie welke massa's van het onbekende materiaal gaat meten.
Bespreek ook hoeveel water er gebruikt gaat worden.
Bepaal de begintemperaturen.
Hevel het aantal afgesproken massa's in de maatbeker. 
Roer voorzicht zodat de temperatuur homogeen is.
Noteer de hoogste gemeten temperatuur, dit is $T_e$.
Wissel de metingen uit met de andere groepen en voer de data-analyse uit.

```{note}
Hieronder staat een voorbeeld hoe je een grafiek moet opslaan en vervolgens in je document kunt oproepen.
Het voorbeeld kun je natuurlijk straks verwijderen.
```

# Resultaten


In [9]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.optimize import curve_fit

c_water = 4186
T_bad = np.mean([69,67,69,67,67,69])
m_blok = 0.050
m_water = np.array([229.1,307.3,218,137.5,331.08,298.8])/1000
T_begin = np.array([22.2,20.3,20.2,21.2,23.5,20.6])
massa_materiaal = np.array([0.05,0.10,0.25,0.20,0.30,0.25])
T_eind = np.array([24.8,21.9,22.5,23.1,24.2,21.6])

c = []

for i in range(len(massa_materiaal)):
    Q_water = m_water[i] * c_water * (T_eind[i] - T_begin[i])
    c_materiaal = Q_water / (massa_materiaal[i] * (T_bad - T_eind[i]))
    c.append(c_materiaal)

for i in range(len(c)):
    print("Specifieke warmte capaciteiten materiaal:", c[i],"J/kgK")
print("Gemiddelde specifieke warmte capaciteit materiaal:", np.mean(c),"J/kgK")

Specifieke warmte capaciteiten materiaal: 1154.3670185185192 J/kgK
Specifieke warmte capaciteiten materiaal: 446.45823861171306 J/kgK
Specifieke warmte capaciteiten materiaal: 184.51520000000005 J/kgK
Specifieke warmte capaciteiten materiaal: 121.78090200445449 J/kgK
Specifieke warmte capaciteiten materiaal: 73.83033607305929 J/kgK
Specifieke warmte capaciteiten materiaal: 107.82558620689656 J/kgK
Gemiddelde specifieke warmte capaciteit materiaal: 348.1295469024405 J/kgK


# Discussie en conclusie

De berekende soortelijke warmte van het onbekende materiaal varieert sterk van 74 J/kgK tot 1154 J/kgK. Het gemiddelde van de metingen komt uit op 348 J/kgK. De spreiding van de metingen is groot, wat kan duiden op experimentele fouten. De fouten kunnen zijn veroorzaakt door warmteverlies aan de omgeving, onnauwkeurige temperatuurmetingen, of onjuiste massa's van het onbekende materiaal. Een systematische fout van de calorimeter zelf kan ook een rol spelen.

Het doel van dit experiment was om de soortelijke warmte van een onbekend materiaal te bepalen. Ondanks de variatie in de resultaten, geeft het gemiddelde een redelijke schatting van de soortelijke warmte. Voor toekomstige experimenten zou het nuttig zijn om de meetmethode te verbeteren, bijvoorbeeld door betere isolatie van de calorimeter en nauwkeurigere meetinstrumenten te gebruiken.